src/components/Auth/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo | null,
+  removeTodo: (param: number | null) => void,
+  isAdding: boolean,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  removeTodo,
+  isAdding,
+}) => {
+  return (
+    <div
+      key={todo?.id}
+      data-cy="Todo"
+      className={classNames(
+        'todo',
+        {
+          completed: todo?.completed,
+        },
+      )}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          defaultChecked
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">{todo?.title}</span>
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDeleteButton"
+        onClick={() => removeTodo(todo?.id || -1)}
+      >
+        ×
+      </button>
+
+      {(isAdding && todo?.id === 0) && (
```

Comment:
```suggestion
      {isLoading && (
```
const isLoading = isAdding && todo?.id === 0